### ISSUE TITLE
How to console.log() from .evaluate() to Node?

### ISSUE THREAD

**Issue by moroshko** [segmentio/nightmare#504]
Is it possible to `console.log()` from `evaluate()` to Node context?

I tried the following, but it doesn't work:

```js
nightmare
  .on('console', () => {
    console.log('yo!');   // Why this is not triggered?
  })
  .evaluate(() => {
    console.log("I'd like to print this out in Node context");
    return 'something';
  })
```

**Comment by rosshinkley** [comment]
Changes were made in 2.1.0 to consolidate how logging works, but the template for evaluating client-side javascript didn't get updated.  This is fixed in #476.

What you have there would work fine if you used `console.warn`, say.

**Comment by moroshko** [comment]
Thanks a lot!
